ex1/exercise_1_x_mix_drix.py

```diff
@@ -67,40 +77,43 @@ def __init__(self, user_input_generator: Generator, ):
         self._board = XOBoard.empty_board()
 
     def change_player(self):
-        """
-        Changes the current player.
-        """
-        pass
+        if self._current_player == PlayerToken.X:
+            self._current_player = PlayerToken.O
+        else:
+            self._current_player = PlayerToken.X
 
     def current_player(self) -> PlayerToken:
-        """
-        Returns the token of the current player.
-        """
-        pass
+        return self._current_player
 
     def get_winner(self) -> PlayerToken:
-        """
-        Returns the token of the winning player if any of the player won.
-        Otherwise, returns None.
-        """
-        pass
+        if self._board.is_winning(X):
+            return X
+        elif self._board.is_winning(O):
+            return O
```

Comment:
For the sake of exercise, implement this function with a `for` loop.
A loop with two simple iterations is not always better - but it is still a good exercise.